platform/darwin/src/MGLLocationManager.h

```diff
@@ -0,0 +1,174 @@
+#import <Foundation/Foundation.h>
+#import <CoreLocation/CoreLocation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@protocol MGLLocationManagerDelegate;
+
+/**
+ The `MGLLocationManager` protocol defines a set of methods that a class must
+ implement in order to serve as the location manager of an `MGLMapView`. A location
+ manager is responsible for notifying the map view about location-related events,
+ such as a change in the user’s location. This protocol is similar to the
+ Core Location framework’s `CLLocationManager` class, but your implementation
+ does not need to be based on `CLLocationManager`.
+ 
+ */
+@protocol MGLLocationManager <NSObject>
```

Comment:
Consider adding this protocol and MGLLocationManagerDelegate to the jazzy table of contents.